ext/NDBM_File/Makefile.PL

```diff
@@ -1,7 +1,16 @@
+use strict;
 use ExtUtils::MakeMaker;
+
+our @LIBS;
+if( $^O eq 'msys' ) {
+    @LIBS = ("-lgdbm -lgdbm_compat");
+} else {
+    @LIBS = ("-L/usr/local/lib -lndbm", "-ldbm -lucb");
+};
+
 WriteMakefile(
     NAME	=> 'NDBM_File',
-    LIBS => ["-L/usr/local/lib -lndbm", "-ldbm -lucb"],
```

Comment:
This is not how we typically do this. Adding `"-lgdbm -lgdbm_compat"` as a third possible value for `LIBS` would be.

That said, I'm not sure we should do this at all. Instead of abusing gdbm's compat layer, such a perl should just use `GDBM_File`